modules/installation-creating-aws-worker.adoc

```diff
@@ -105,76 +105,8 @@ directory. This value is the long string with the format
 <14> Specify the `WorkerInstanceProfile` parameter value from the output of
 the CloudFormation template for the security group and roles.
 <15> The type of AWS instance to use for the control plane machines.
```

Comment:
this line should be `The type of AWS instance to use for the compute machines.`
@mtulio can you help fix this? Thanks. 

Others LGTM.

Reply:
@yunjiang29 this line was also fixed. Thanks to report it.